packages/react/src/contexts/SuspensiveDevModeContext.spec.tsx

```diff
@@ -48,4 +33,16 @@ describe('SuspensiveDevMode', () => {
     suspensiveDevMode.off()
     expect(subscriber.notifiedCount).toBe(2)
   })
+
+  it("shouldn't be get by useSuspensiveDevModeContext if no SuspensiveProvider in parent", () => {
+    const { result } = renderHook(() => useContext(DevModeContext))
+    expect(result.current).toBeNull()
+  })
+
+  it('should be get by useSuspensiveDevModeContext if SuspensiveProvider in parent', () => {
+    const { result } = renderHook(() => useContext(DevModeContext), {
+      wrapper: (props) => <SuspensiveProvider {...props} value={new Suspensive()} />,
+    })
+    expect(result.current).toBeInstanceOf(SuspensiveDevMode)
+  })
```

Comment:
How about using `describe`? for instance:

```suggestion
  describe("DevModeContext", () => {
    it("returns null when no SuspensiveProvider is present", () => {
      const { result } = renderHook(() => useContext(DevModeContext))
      expect(result.current).toBeNull()
    })

    it("returns an instance of SuspensiveDevMode when wrapped with SuspensiveProvider", () => {
      const { result } = renderHook(() => useContext(DevModeContext), {
        wrapper: (props) => (
          <SuspensiveProvider {...props} value={new Suspensive()} />
        ),
      })
      expect(result.current).toBeInstanceOf(SuspensiveDevMode)
    })
  })
```

Reply:
Cool